addons/common/XEH_preInit.sqf

```diff
@@ -71,4 +71,6 @@ activateAddons GVAR(addons);
     }];
 }] call CBA_fnc_addClassEventHandler;
 
+GVAR(magNamespace) = call CBA_fnc_createNamespace; // namespace used for CBA_fnc_compatibleMagazines
```

Comment:
Now the function no longer works in preInit from components loaded before cba_common.

Reply:
The caching at least, so I guess it's fine in a way?

Reply:
```suggestion
```